zerver/migrations/0760_realm_send_channel_events_messages.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 5.2.7 on 2025-11-08 13:55
+
+from django.db import migrations, models
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def set_send_channel_events_messages_for_existing_organizations(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    Realm = apps.get_model("zerver", "Realm")
+    Realm.objects.all().update(send_channel_events_messages=True)
```

Comment:
Please add a comment explaining the reasoning here: we're preserving the `True`  prior behavior for existing organizations, while setting the default for new organizations to `False`.